modules/command-engine/core/src/main/scala/surge/core/Context.scala

```diff
@@ -2,11 +2,13 @@
 
 package surge.core
 
+import akka.actor.ActorRef
+
 import scala.concurrent.ExecutionContext
-import scala.concurrent.ExecutionContext.global
 
-object Context {
-  val noop: Context = new Context()
+private[surge] object Context {
```

Comment:
If these are both going to be private now (which makes sense to me for the core one) we can move them to the `surge.internal` package